xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -3452,13 +3452,8 @@ fuse_release(xlator_t *this, fuse_in_header_t *finh, void *msg,
 
     fd_close(state->fd);
 
-    fuse_fd_ctx_destroy(this, state->fd);
-    fd_unref(fd);
```

Comment:
Should we really delete the unrefing of the fd. I guess we have a ref on the fd hold by fuse that should be unrefed in fuse_release.